randovania/games/prime2/patcher/claris_randomizer.py

```diff
@@ -36,7 +36,11 @@ def _get_randomizer_folder() -> Path:
 
 
 def _get_randomizer_path() -> Path:
-    return _get_randomizer_folder().joinpath("Randomizer.exe")
+    nativepath = _get_randomizer_folder().joinpath("randomizer")
+    if nativepath.is_file():
```

Comment:
I prefer if we'd be able to check for the situation where the different path is needed, and then use the different path.
So if when running from app image uses this path, the if should check for that.

The reasoning means we don't hide errors. If the file isn't where we expect, I want the error message to yell the correct file path that is wrong.